stellar_sdk/soroban_server.py

```diff
@@ -236,6 +237,43 @@ def send_transaction(
         )
         return self._post(request, SendTransactionResponse)
 
+    def poll_transaction(
+        self,
+        transaction_hash: str,
+        attempts: int = 30,
+        sleep_strategy: SleepStrategy = BasicSleepStrategy,
+    ) -> GetTransactionResponse:
+        """Poll for a particular transaction with certain parameters.
+
+        After submitting a transaction, clients can use this to poll for transaction completion and return a definitive state of success or failure.
+
+        :param transaction_hash: The hash of the transaction to poll for.
+        :type transaction_hash: str
+        :param attempts: The number of attempts to make before returning the last-seen status, defaults to 30.
+        :type attempts: int, optional
+        :param sleep_strategy: The amount of time to wait for between each attempt, defaults to 1 second between each attempt.
+        :type sleep_strategy: SleepStrategy, optional
+        :return: A :class:`GetTransactionResponse <stellar_sdk.soroban_rpc.GetTransactionResponse>` response object after a "found" response, (which may be success or failure) or the last response obtained after polling the maximum number of specified attempts.
+        :rtype: GetTransactionResponse
+        :raises: :exc:`SorobanRpcErrorResponse <stellar_sdk.exceptions.SorobanRpcErrorResponse>` - If the Soroban-RPC instance returns an error response.
```

Comment:
I'm realizing that some plugin in my editor, which provides a "create docstring" function when I type `"""`, has done these parameters and types a little differently than the other existing functions here. Do you want me to adapt these docstring params/types to be more like the others?